spinn_front_end_common/utility_models/data_speed_up_packet_gatherer_machine_vertex.py

```diff
@@ -218,6 +357,23 @@ def generate_data_specification(
         iptag.port = self._connection.local_port
         spec.write_value(iptag.tag)
 
+        # write mc chip key map
+        spec.switch_write_focus(
+            self.DATA_REGIONS.DATA_IN_CHIP_TO_KEY_SPACE.value)
+        chips_on_board = list(machine.get_chips_on_board(
```

Comment:
Do we need to convert this into a list?

Reply:
yes, as len of a iterator dont work. (line 367). so listing it allows len to work correctly